### ISSUE TITLE
Default Directory Structure for Each Type of Files  

### ISSUE THREAD

**Issue by sirinath** [11ty/eleventy#957]
It will be informative if the default directory structure with file types and look up and processing order is added to the documentation.

**Comment by josephborg** [comment]
agreed!! since 11ty is a file-based static CMS, the directory structure should be well documented. Preferably with notes on what can be moved and what cannot. I still cannot get a good picture of the directory layout from the documentation and how it can be reconfigured.

**Comment by nhoizey** [comment]
I think there is no "default directory structure" for contents, apart from `_data` for global data files.

There are [some defaults](https://www.11ty.dev/docs/config/#configuration-options) for other "components" like includes, default build folder, etc., but these are not content.
